apps/lockfile-explorer-web/src/containers/LockfileEntryDetailsView/index.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import styles from './styles.scss';
+import appStyles from '../../appstyles.scss';
+import { IDependencyType, LockfileDependency } from '../../parsing/LockfileDependency';
+import { useAppDispatch, useAppSelector } from '../../store/hooks';
+import { pushToStack, selectCurrentEntry } from '../../store/slices/entrySlice';
+import { ReactNull } from '../../types/ReactNull';
+import { LockfileEntry } from '../../parsing/LockfileEntry';
+
+enum InfluencerTypes {
+  DETERMINANT,
+  TRANSITIVE_REFERRER
+}
+
+interface IInfluencerType {
+  entry: LockfileEntry;
+  type: InfluencerTypes;
+}
+
+export const LockfileEntryDetailsView = (): JSX.Element | ReactNull => {
+  const selectedEntry = useAppSelector(selectCurrentEntry);
+  const dispatch = useAppDispatch();
+
+  const [inspectDep, setInspectDep] = useState<LockfileDependency | null>(null);
```

Comment:
Spell out the word "Dependency" instead of "Dep".  Rush's code convention is to use full English words instead of abbreviations.  (The goal is that a complete stranger should be able to understand the code easily, even if they have no idea what this app does, even if they never used React before. Maybe their purpose in modifying this code something totally unrelated, like upgrading Webpack across many projects.)

Reply:
changed